packages/react-components/react-storybook-addon-codesandbox/src/sandbox-factory.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { ParametersExtension, StoryContext } from './types';
+import { addDemoActionButton } from './sandbox-factory';
+describe(`sandbox-factory`, () => {
```

Comment:
not the best test in the world. ideal would be to have e2e with playwright , but even then we would need to hijack submission. so this is good for now IMO.